utils/get_data.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+ LizardDownloader
+                                 A QGIS plugin
+ This plug-in helps with downloading data fromLizard in QGIS.
+                              -------------------
+        begin                : 2017-01-25
+        git sha              : $Format:%H$
+        copyright            : (C) 2017 by Madeleine van Winkel
+        email                : madeleine.vanwinkel@nelen-schuurmans.nl
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+# Import
```

Comment:
``# Import`` is redundant
License information is also redundant (already in the LICENSE file in the project root).

 I think it's better to remove these, and just have a docstring that describes the module.